.github/workflows/phpcs.yml

```diff
@@ -1,44 +1,20 @@
 name: 'PHPCS'
 on: [pull_request]
+
 jobs:
   conditional:
-    if: "!contains(github.event.pull_request.body, '[skip-phpcs]')"
-    runs-on: ubuntu-latest
+    # MODIFIED: Using reusable php-change-detector workflow
```

Comment:
This line is a left-over from debug / generation, it seems.

Reply:
Yeah, I vibe-coded a lot of this and the AI was comment-happy.